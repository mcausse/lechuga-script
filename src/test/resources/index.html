<html>
<head>
    <title><%= title %></title>
</head>
<body>
    <h1><%= title %></h1>
    <ul>
        <% (for dog dogs %>
        <li><%= (dog :getId) %> - <%= (dog :getName) %></li>
        <% ) %>
    </ul>
</body>
</html>
