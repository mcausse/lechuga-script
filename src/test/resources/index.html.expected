<html>
<head>
    <title>JOU</title>
</head>
<body>
    <h1>JOU</h1>
    <ul>

        <li>123 - chucho</li>

        <li>456 - faria</li>

    </ul>
</body>
</html>